contracts/AcceleratingDistributor.sol

```diff
@@ -97,9 +97,14 @@ contract AcceleratingDistributor is Testable, ReentrancyGuard, Pausable, Ownable
         uint256 maxMultiplier,
         uint256 secondsToMaxMultiplier
     ) public onlyOwner {
+        // Validate input to ensure system stability and avoid unexpected behavior.
         // Because of the way balances are managed, the staked token cannot be the reward token. Otherwise, reward
         // payouts could eat into user balances.
         require(stakedToken != address(rewardToken), "Staked token is reward token");
+        require(maxMultiplier > 1e18, "maxMultiplier be larger than 1");
```

Comment:
This seems to limit flexibility? Is there a reason we want to _not allow_ emissions to decrease over time? It seems somewhat arbitrary unless it causes some problem in the contract.

Reply:
ye, I agree it's limiting. I left a comment to the same effect in the solution part of the PR description.

I think I'm going to remove this and add a comment in the code that if less than one will eat away at your balance. will specifically point this out to OZ that we did not totally implement all restrictions. 

you good with this?

Reply:
Yep! +1